tests/python/pants_test/reporting/test_reporting.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from pants.goal.run_tracker import RunTracker
+from pants.reporting.reporting import Reporting
+from pants_test.test_base import TestBase
+
+
+class ReportingTest(TestBase):
+
+  # Options for Zipkin tracing
+  trace_id = "0123456789abcdef"
+  parent_id = "0123456789abcdef"
```

Comment:
Let's use different values for trace_id and parent_id, so that we can make sure we didn't confuse them in our real code